pkg/operator/starter.go

```diff
@@ -0,0 +1,95 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cert-manager-operator/pkg/controller/deployment"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned"
+	configinformers "github.com/openshift/client-go/config/informers/externalversions"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/openshift/library-go/pkg/operator/genericoperatorclient"
+	"github.com/openshift/library-go/pkg/operator/status"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	TargetNamespace   = "openshift-cert-manager"
+	OperatorNamespace = "openshift-cert-manager-operator"
```

Comment:
suggestion: move all constants into a single folder, like `api` or something similar to KAS-o's https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/operatorclient/interfaces.go

Reply:
sounds good, i think i copy-pasted this from somewhere